boards/nxp/common/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NXP_SPSDK_IMAGE
+	bool "Generate bootable firmware image with SPSDK"
```

Comment:
Is this something this SoC will use, i.e. on any board? If so, should it go at soc level rather than board?

Reply:
Sure